pom.xml

```diff
@@ -637,6 +639,18 @@ under the License.
 	</dependencyManagement>
 
 	<profiles>
+		<profile>
```

Comment:
This profile is not necessary but simplifies the way to run tests for annotated MiniClusters again the new scheduler.
We can run "mvn verify -Dscheduler.type=ng -Dtest.groups=org.apache.flink.testutils.junit.category.AlsoRunWithSchedulerNG" instead.
And to find out whether all the MiniCluster tests(even not annotated with `AlsoRunWithSchedulerNG`) can pass against the new scheduler, we can run "mvn verify -Dscheduler.type=ng".

Reply:
I think the profile is fine.